central/compliance/service/service_impl.go

```diff
@@ -127,9 +127,19 @@ func (s *serviceImpl) GetAggregatedResults(ctx context.Context, request *v1.Comp
 		return nil, err
 	}
 
+	filteredSources := sources[:0]
+	for _, source := range sources {
+		config, exsist, errConfig := s.complianceDataStore.GetConfig(ctx, source.GetStandardId())
```

Comment:
Please fix the typo. `exsist` should be `exist`

and `errConfig` should just be `err`

Reply:
@theencee  There is already err declared in the function